strings/strings.json

```diff
@@ -7552,5 +7552,25 @@
   "HierarchicalFacetValuePathPrefix": {
     "en": "in",
     "fr": "dans"
+  },
+  "UsefulnessFeedbackRequest": {
+    "en": "Was this useful?",
+    "fr": "Était-ce utile?"
+  },
+  "UsefulnessFeedbackThankYou": {
+    "en": "Thanks for your comments!",
+    "fr": "Merci pour vos commentaires!"
+  },
```

Comment:
This message is displayed **before** the `Explain why` button is clicked. This is the text as shown in the mock-up.

Do you think a string such as "Thanks for your feedback!" would be more relevant?

Reply:
Yes, I think "Thanks for your feedback!"/"Merci pour votre rétroaction!" would be better, since the end user is not necessarily going to provide further explanations.

Reply:
Perfect, I'll change it then.